README.md

```diff
@@ -126,7 +126,7 @@ require('renamer').setup {
         left = 0,
         bottom = 0,
         right = 0,
-    }
+    },
```

Comment:
Nice catch, we should also add this in the `lua/renamer/init.lua` comments above the `setup()` function and the `doc/renamer.txt` vimdoc file